app/models/bigquery_config.rb

```diff
@@ -0,0 +1,6 @@
+class BigqueryConfig < ApplicationRecord
+  validates :project_id, :dataset, presence: true
+
+  self.primary_key = :data_source_id
```

Comment:
Why do you want to change primary key?
I think it is safe to leave the primary key as default.

Reply:
`bigquery_configs.data_source_id` is unique and not null. I think it can be used as primary key.
You don't prefer? If not, I will fix.